chunking_list/chunking_list.py

```diff
@@ -0,0 +1,13 @@
+def chunk_list(list_1, chunk_size):
```

Comment:
Please use function signatures with type hints for the task description.
P.S.: For `List` and `Any`, use the `typing` module.